filebeat/registrar/registrar.go

```diff
@@ -185,14 +185,14 @@ func (r *Registrar) onEvents(states []file.State) {
 	r.processEventStates(states)
 
 	beforeCount := r.states.Count()
-	cleanedStates := r.states.Cleanup()
+	cleanedStates, pendingClean := r.states.Cleanup()
 	statesCleanup.Add(int64(cleanedStates))
 
 	r.bufferedStateUpdates += len(states)
 
 	logp.Debug("registrar",
-		"Registrar states cleaned up. Before: %d, After: %d",
-		beforeCount, beforeCount-cleanedStates)
+		"Registrar states cleaned up. Before: %d, After: %d, Pending: %d",
```

Comment:
I already see questions coming on what pending cleanup means :-)

Reply:
We can call it 'active'. The number of 'pending' states is all states with TTL > 0.

Following the state changes and TTLs, I found TTL basically has 4 meanings:
- `-2`: zombie state (no prospector/input processing the file)
- `-1`: alive (prospector/input owning the file, but no TTL)
- `0`: done (entry is marked as removable)
- `>0`: subject to timeout

Any better naming?

Currently The 'pending' count is the number of states potentially subject to being timed out. But states `-1` and `>0` indicate the state to be 'active'.